.github/workflows/job_build_windows.yml

```diff
@@ -239,7 +248,8 @@ jobs:
 
             cmake -DPython3_EXECUTABLE="$pythonExecutablePath" -DOpenVINODeveloperPackage_DIR=${{ env.BUILD_DIR }} -S ${{ env.OPENVINO_REPO }}/src/bindings/python -B "$pyBuildDir" &&
```

Comment:
I think we need to explicitly set -DENABLE_PYTHON=ON -DENABLE_WHEEL=ON here and set them to OFF for the main build https://github.com/mryzhov/openvino/blob/ci/install_build_dependencies/.github/workflows/windows_vs2022_release.yml#L66-L67